src/EA.Weee.Web.Tests.Unit/Areas/Aatf/Controller/ManageEvidenceNotesControllerCreateEvidenceNoteTests.cs

```diff
@@ -224,6 +225,50 @@ public async Task CreateEvidenceNotePost_GivenModelIsValid_ShouldBeRedirectToVie
             result.RouteValues.Should().Contain(c => c.Key.Equals("evidenceNoteId"));
         }
 
+        [Fact]
+        public async void IndexGet_GivenValidViewModel_BreadcrumbShouldBeSet()
+        {
+            var organisationName = "Organisation";
+
+            A.CallTo(() => cache.FetchOrganisationName(A<Guid>._)).Returns(organisationName);
+
+            await controller.Index(A.Dummy<Guid>(), A.Dummy<Guid>());
+
+            breadcrumb.ExternalOrganisation.Should().Be(organisationName);
+            breadcrumb.ExternalActivity.Should().Be($"Manage AATF Evidence Notes");
+        }
+
+        [Fact]
+        public async void IndexGet_GivenOrganisationId_ApiShouldBeCalled()
+        {
+            var organisationId = Guid.NewGuid();
+            var aatfId = Guid.NewGuid();
+            var model = new ManageEvidenceNoteViewModel()
+            {
+            };
+
+            var result = await controller.Index(organisationId, aatfId);
+
+            A.CallTo(() => weeeClient.SendAsync(A<string>._, A<GetAatfByIdExternal>.That.Matches(w => w.AatfId == aatfId))).MustHaveHappened(1, Times.Exactly);
+        }
+
+        [Fact]
+        public async void IndexPost_ValidViewModel_PageRedirectsCreateEvidenceNote()
+        {
+            var model = new ManageEvidenceNoteViewModel()
+            {
+                OrganisationId = fixture.Create<Guid>(),
+                AatfId = fixture.Create<Guid>(),
+            };
+
+            var result = await controller.Index(model) as RedirectToRouteResult;
+
+            result.RouteValues["action"].Should().Be("Index");
+            result.RouteValues["controller"].Should().Be("Holding"); // TODO Change to Create Evidence Note
+            result.RouteValues["organisationId"].Should().Be(model.OrganisationId);
+            //result.RouteValues["aatfId"].Should().Be(model.SelectedId); // This will be needed
+        }
+
```

Comment:
Need test to ensure that the ManageEvidenceNoteViewModel is returned to the View